rust/navigator/src/tests.rs

```diff
@@ -1763,360 +5101,1376 @@ fn flow_test_1() {
-    assert!(
-        cut_real_json == expected_json,
-        "Switched to Log from Settings. Expected Log screen, got:\n{}",
-        real_json
-    );
+        let real_json = do_action(Action::NavbarLog, "", "").unwrap();
+        let cut_real_json = cut_public_key(&cut_base58(&cut_identicon(&timeless(&real_json))));
+        let expected_json = r#"{"screen":"Log","screenLabel":"","back":false,"footer":true,"footerButton":"Log","rightButton":"LogRight","screenNameType":"h4","modal":"Empty","alert":"Empty","screenData":{"log":[{"order":1,"timestamp":"**","events":[{"event":"message_signed","payload":{"message":"4c6f72656d20697073756d20646f6c6f722073697420616d65742c20636f6e73656374657475722061646970697363696e6720656c69742c2073656420646f20656975736d6f642074656d706f7220696e6369646964756e74207574206c61626f726520657420646f6c6f7265206d61676e6120616c697175612e20557420656e696d206164206d696e696d2076656e69616d2c2071756973206e6f737472756420657865726369746174696f6e20756c6c616d636f206c61626f726973206e69736920757420616c697175697020657820656120636f6d6d6f646f20636f6e7365717561742e2044756973206175746520697275726520646f6c6f7220696e20726570726568656e646572697420696e20766f6c7570746174652076656c697420657373652063696c6c756d20646f6c6f726520657520667567696174206e756c6c612070617269617475722e204578636570746575722073696e74206f6363616563617420637570696461746174206e6f6e2070726f6964656e742c2073756e7420696e2063756c706120717569206f666669636961206465736572756e74206d6f6c6c697420616e696d20696420657374206c61626f72756d2e","network_name":"westend","signed_by":{"public_key":"**","identicon":"**","encryption":"sr25519"},"user_comment":"Pepper tries better"}}]},{"order":0,"timestamp":"**","events":[{"event":"history_cleared"}]}],"total_entries":2},"modalData":{},"alertData":{}}"#;
+        assert!(
+            cut_real_json == expected_json,
+            "Switched to Log from Settings. Expected Log screen, got:\n{}",
+            real_json
+        );
 
-    // What if the database is not initiated properly? This should have been a separate test, but mutex.
-    populate_cold_nav_test(dbname).unwrap(); // no init after population
-    let real_json = do_action("NavbarSettings", "", "");
-    let expected_json = r#"{"screen":"Settings","screenLabel":"","back":false,"footer":true,"footerButton":"Settings","rightButton":"None","screenNameType":"h4","modal":"Empty","alert":"Empty","screenData":{"error":"Could not find general verifier."},"modalData":{},"alertData":{}}"#;
-    assert!(real_json == expected_json, "Switched to Settings from Log with non-initiated database. Expected Settings screen with error on screen, and no alerts (we should still allow to reset Signer), got:\n{}", real_json);
+        // What if the database is not initiated properly? This should have been a separate test, but mutex.
+        populate_cold_nav_test(dbname).unwrap(); // no init after population
```

Comment:
This piece should be back. Having uninitiated database was quite a common problem (for users who did not reset Signer before updating as instructed, but still), we need this test to make sure they have a way out.